### DISCUSSION TITLE
adding to existing project

### DISCUSSION THREAD

**Discussion by rot3r-git** [alex8088/electron-vite#82] [Q&A]
i just follow the instruction and use : npm i electron-vite -D in a existing project but i got error "Cannot find module 'electron/package.json'".
is there any further instruction that i have to follow?

**Comment by alex8088** [comment]
you need to install `electron`.

**Reply by rot3r-git** [reply]
can you please describe more or share a link that i have to follow?

**Reply by alex8088** [reply]
https://github.com/alex8088/electron-vite-boilerplate
